testplan/testing/multitest/driver/base.py

```diff
@@ -90,7 +91,10 @@ class Driver(Resource):
     :type install_files: ``List[Union[str, tuple]]``
     :param timeout: Timeout duration for status condition check.
     :type timeout: ``int``
-    :param logname: Driver logfile name.
+    :param logname: Driver logfile name. When a binary started by driver makes
```

Comment:
Driver logfile base name, in which testplan will look for ``log_regexps`` as driver start-up condition. Default to (driver_name.log?) under (app_path?).